src/transformers/generation/configuration_utils.py

```diff
@@ -227,6 +227,20 @@ class GenerationConfig(PushToHubMixin):
         decoder_start_token_id (`int`, *optional*):
             If an encoder-decoder model starts decoding with a different token than *bos*, the id of that token.
 
+        > Generation parameters exclusive to [assistant generation](https://arxiv.org/abs/2211.17192)
+
+        max_assistant_tokens (`int`, *optional*, defaults to 5):
```

Comment:
Perhaps we can take the chance to give a better name to this variable: `assistant_tokens` or similar. `max_assistant_tokens` implies that the assistant will never cross this limit but, as we can see in `max_assistant_tokens_schedule` (which should also be renamed accordingly), that is not true :)

Reply:
Sounds good!